django/template/response.py

```diff
@@ -12,34 +12,39 @@ class SimpleTemplateResponse(HttpResponse):
 
     def __init__(self, template, context=None, content_type=None, status=None,
                  charset=None, using=None):
-        # It would seem obvious to call these next two members 'template' and
-        # 'context', but those names are reserved as part of the test Client
-        # API. To avoid the name collision, we use different names.
+        """
+        It would seem obvious to call these next two members 'template' and
+        'context', but those names are reserved as part of the test Client
+        API. To avoid the name collision, we use different names.
+        """
```

Comment:
This converts the inline comment to the docstring for the method. That's not correct. Please revert.